### DISCUSSION TITLE
Is there lightmode?

### DISCUSSION THREAD

**Discussion by VamsiPulipati** [monkeytypegame/monkeytype#5364] [Ideas]
are there good lightmode themes

**Comment by fehmer** [comment]
yes.

**Comment by kumail-r** [comment]
"Good" is very subjective but you can take a look on your own by either pressing `esc` and typing `theme` and using the up/down arrow keys to cycle through the list of available themes, or by going into the corresponding page in [settings](https://monkeytype.com/settings#group_theme). You can even create a custom theme or override color values if you aren't satisfied by the available presets.
